Use dummy MIDI implementation instead of ALSA specific one.
Do not link against asound (ALSA).

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -98,7 +98,7 @@ else ()
     set(BZC_PROJECT_LIBS ${BZC_PROJECT_LIBS} Threads::Threads)
   endif ()
 
-  set(PLATFORM_LIBS GL asound fontconfig)
+  set(PLATFORM_LIBS GL fontconfig)
 endif ()
 
 ##############################################################################
@@ -332,7 +332,7 @@ elseif (UNIX)
   set(BZC_PLATFORM_SRCS
     ${CMAKE_SOURCE_DIR}/src/platform_glfw/Renderer.cpp
     ${CMAKE_SOURCE_DIR}/src/platform_common/FFT.cpp
-    ${CMAKE_SOURCE_DIR}/src/platform_x11/MIDI.cpp
+    ${CMAKE_SOURCE_DIR}/src/platform_common/MIDIDummy.cpp
     ${CMAKE_SOURCE_DIR}/src/platform_x11/Misc.cpp
     ${CMAKE_SOURCE_DIR}/src/platform_x11/SetupDialog.cpp
     ${CMAKE_SOURCE_DIR}/src/platform_x11/Timer.cpp
